docs/multi-stage-query/known-issues.md

```diff
@@ -40,7 +40,7 @@ an [UnknownError](./reference.md#error-codes) with a message including "No space
 - SELECT from a Druid datasource does not include unpublished real-time data.
 
 - GROUPING SETS and UNION ALL are not implemented. Queries using these features return a
-  [QueryNotSupported](reference.md#error-codes) error.
+  [QueryNotSupported](reference.md#error-codes) error. For some COUNT DISTINCT queries, the planner attempts to use GROUPING SETS, which causes the following exception: `Must not have 'subtotalsSpec'`.
```

Comment:
I feel that from a user-facing PoV, this is a different known issue from the one about `GROUPING SETS` and `UNION ALL`, so it deserves its own line. (On a technical level, the reason for certain `COUNT DISTINCT` queries not being supported is related to their internal usage of `GROUPING SETS`, but that isn't super user-visible.)

other nit: I would say this causes an exception _mentioning_ "Must not have 'subtotalsSpec'", rather than causing that exception specifically. That's because the top-level exception is QueryNotSupported; the "Must not have 'subtotalsSpec'" is reported in one of the causes.

Reply:
Thanks for the clarification! I'll push a new commit with some changes